life-sync.cabal

```diff
@@ -47,57 +83,35 @@ library
                      , path
                      , path-io
                      , process
-                     , relude >= 0.4.0
                      , text >= 1.2
-                     , tomland ^>= 0.3
-
-  ghc-options:         -Wall
-  default-language:    Haskell2010
-
-  default-extensions:  GeneralizedNewtypeDeriving
-                       LambdaCase
-                       OverloadedStrings
-                       RecordWildCards
-                       ScopedTypeVariables
-                       TypeApplications
+                     , tomland ^>= 1.2.1.0
 
 executable life
+  import:              common-options
   hs-source-dirs:      app
   main-is:             Main.hs
   other-modules:       Options
-  ghc-options:         -Wall -threaded -rtsopts -with-rtsopts=-N
+  ghc-options:         -threaded -rtsopts -with-rtsopts=-N
 
-  build-depends:       base-noprelude
-                     , containers
+  build-depends:       containers
```

Comment:
What do you think about having the dependancy on the library first in the executable stanzas? In this case to put `life-sync` on the first place?